app/scripts/app.js

```diff
@@ -7,9 +7,14 @@
       });
 
       $stateProvider
+        .state('main', {
+          url: '/',
+          templateUrl: '/templates/main.html'
+        });
+
   }
 
   angular
-    .module('blocChat',['ui.router'])
+    .module('blocChat', ['ui.router'])
```

Comment:
Add firebase here: `.module('blocChat', ['ui.router', 'firebase'])`.

In future code, you'll be able to inject Firebase (probably `$firebaseArray`) into your controllers. For now, this should be all of the setup required for what you have. It might be worth it to create a MainCtrl to test the injection works. You'll need to associate the MainCtrl with the main state a couple of lines above this one.

Hope this helps!